src/bundles/SellerRegistration/components/DocumentsForm/DocumentsForm.js

```diff
@@ -352,17 +351,6 @@ class DocumentsForm extends BaseForm {
                                 </div>
                             )
                         })}
-                        {documentsForm.documents.workers && documentsForm.documents.workers.filename ?
-                          '' :
-                          <CheckboxDetailsField
```

Comment:
Remove the import for CheckboxDetailsField since it is not used anymore

Reply:
CheckboxDetailsField was removed